manifests/init.pp

```diff
@@ -200,17 +222,78 @@
     }
   }
 
-  if $pulp {
-    validate_present($pulp_oauth_secret)
+  class { 'capsule::install': } ~>
+  class { 'certs::foreman_proxy':
+    hostname => $capsule_fqdn,
+    require  => Package['foreman-proxy'],
+    before   => Service['foreman-proxy'],
+  } ~>
+  class { 'certs::katello':
+    deployment_url => $capsule::rhsm_url,
+    rhsm_port      => $capsule::rhsm_port
   }
```

Comment:
Disregard, it was removed - needed to update that module in my katello-installer